src/v2/components/Indicators/LoadingDots.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { StyleProp, View, ViewStyle } from 'react-native';
+
+import { LoadingDot } from './LoadingDot';
+
+import { useTheme } from '../../contexts/themeContext/ThemeContext';
+
+type Props = {
+  diameter?: number;
+  duration?: number;
+  numberOfDots?: number;
+  spacing?: number;
+  style?: StyleProp<ViewStyle>;
+};
+
+export const LoadingDots: React.FC<Props> = (props) => {
+  const {
+    diameter = 4,
+    duration = 1500,
+    numberOfDots = 3,
+    spacing: spacingProp,
+    style,
+  } = props;
+
+  const {
+    theme: {
+      loadingDots: { container, spacing },
+    },
+  } = useTheme();
+
+  const halfSpacing = spacingProp ? spacingProp / 2 : spacing / 2;
+  const offsetLength = duration / numberOfDots;
+
+  return (
+    <View style={[{ flexDirection: 'row' }, container, style]}>
+      {[...Array(numberOfDots).keys()].map((_item, index) => (
```

Comment:
I did this instead for the channel list skeleton. `Array.from(Array(numberOfSkeletons)).map((_, index) => (`

which style do we want to go with?